### PR TITLE
#1654: Show and Hide the live demos based on current tab

### LINKED ISSUE #1654: Pop-up toolbox persists
When there are enough buttons or the window is small enough, the ellipse (...) menu appears in the toolbar. Clicking on it will show a pop-up box by default that contains the remaining toolbar buttons. When the tinymce field is hidden, this box incorrectly persists.

Reproducible on the basic example https://www.tiny.cloud/docs/demo/basic-example/#
1. Make the window small enough that the toolbar displays an ellipse (...) menu.
2. Click the ellipse menu.
3. Click the "HTML" tab
4. Scroll down and the pop-up box from the menu is still visible despite the tinymce text box being hidden.

The pop-up toolbox should be hidden when the tinymce component is hidden. I also noticed that the pop-up does not close when the user clicks away, but I'm not sure if that is the intended behavior. As a workaround in the meantime, I set toolbar_mode to 'sliding' in my tinymce settings.

I first noticed this issue in Chrome on version 5.2.2

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: https://github.com/tinymce/tinymce-docs/issues/1654

Description of Changes:
* Show or hide live demos based on currently visible tab
Fixes #1654 

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Review by EkimChau — APPROVED** [review]

**Review by metricjs — APPROVED** [review]

**Review by lnewson — CHANGES_REQUESTED** [review]
Won't work with tinydrive standalone pages and also will have issues if more than one editor is on the page. I'll create a patch later today.

**Review by lnewson — APPROVED** [review]

**Review by metricjs — APPROVED** [review]

File: _includes/live-demo.html [hunk 1]
```diff
@@ -222,17 +222,34 @@
...
 
+    var lastTabName = '{{ initialTab }}';
     tabs.forEach(function(t) {
       t.tab.onclick = function(e) {
-        tabs.forEach(function(tt) {
-          tt.pane.style.display = t === tt ? 'block' : 'none';
-          tt.tab.className = t === tt ? 'live_demo_tab_selected' : 'live_demo_tab_deselected';
-        });
+        if (lastTabName !== t.name) {
+          tabs.forEach(function(tt) {
```
Comment (ashfordneil) [inline _includes/live-demo.html:235]: Can we change `tt` to something else?

**Review by ashfordneil — APPROVED** [review]

File: _includes/live-demo.html [hunk 2]
```diff
@@ -222,17 +222,34 @@
     /* TODO: */
```
[the remark this answers is not in the document]
Comment (tylerkelly13) [reply _includes/live-demo.html:222]: always. :)

[the remark this answers is not in the document]
Comment (lnewson) [reply _includes/live-demo.html:222]: I honestly have no idea what this is meant to be for...

**Pull request merged**